src/Models/Product.java

```diff
@@ -1,20 +1,30 @@
 package Models;
 
 public class Product {
+    private int id;
     private String name;
     private String type;
     private float price;
     private int qty_on_hand;
     private String description;
 
-    public Product(String name, String type, float price, int qty_on_hand, String description) {
+    public Product(int id, String name, String type, float price, int qty_on_hand, String description) {
```

Comment:
I think there should be another constructor or two implemented for this that allows for the id not to be used. I think forcing the id to -1 might be a mistake. One solutions is just a constructor with no params and then just set what you need with getters and setters.

Reply:
yeah that's a good point, I'll refactor it that way rather than making it assign a bogus id that won't be used.